Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/LobbyViewModel.cs

```diff
@@ -67,9 +67,16 @@ public class LobbyViewModel : BaseViewModel {
             gameModel.Id = this.currentUser.InviteKey.GameId;
 
             webSocketService = new WebSocketService(gameModel.Id);
-            Task.Run(async () => await StartSocket());
+
             Task.Run(async () => await LoadUsers());
-            Task.Run(async () => await CheckForStatus());
+            Task.Run(async () => {
+                IsLoading = true;
```

Comment:
Misschien kan IsLoading al op true gezet worden boven `Task.Run(async () => await LoadUsers());`?

Reply:
Nee ik wil pas het laadscherm weglaten als echt alles is ingeladen. Dat maakt het visueel wat duidelijker voor de gebruiker dat het spel nog wordt klaargezet. Anders wordt ook het scherm telkens geupdate door allerlei wijzingen, terwijl euhm ja.

Reply:
Maar met mijn suggestie wordt het laad scherm als het goed is juist iets eerder weergegeven, en veranderd het moment waarop het weg gaat niet.